config/site_settings.yml

```diff
@@ -3523,3 +3523,6 @@ experimental:
     type: group_list
     list_type: compact
     area: "group_permissions"
+  use_overhauled_theme_color_palette:
```

Comment:
The "overhauled" in the name is kind of an artefact of its time. It doesn't quite describe the setting.

How about `use_two_tone_color_palette`?

Reply:
My intent is to use this setting as a feature flag for the whole theme and color palette overhaul project, which includes several other changes. We'll also drop this setting once the project is complete and rolled out to sites, so it won't be very long-lived.